### ISSUE TITLE
Typo on the page example

### ISSUE THREAD

**Issue by naceurCRAAG** [SciML/NeuralPDE.jl#664]
Dear Chris

Could you please correct a typo found on the example of the homepage here
https://github.com/SciML/NeuralPDE.jl
in the script on the 19th line 
chain = Lux.Chain(Dense(dim, 16, Lux.σ), Dense(16, 16, Flux.σ), Dense(16, 1))

replace Flux with Lux

and also add at the top of the example 
using OptimizationOptimisers
since we are using ADAM as a gradient descent

I made this changes and the example works 

Have A Nice Day

Naceur

**Comment by xtalax** [comment]
Hey Naceur, why don't you create a pull request to fix this?

Here are some helpful instructions, in part generated by ChatGPT:

Editing the README of a GitHub repository that you don't own requires you to create a fork of the repository, make the desired changes to the README in your forked repository, and then submit a pull request to the original repository. Here are the step-by-step instructions:

1. Visit the GitHub repository that you want to edit the README of. (This one)
2. Click on the "Fork" button on the top right-hand corner of the repository page. This will create a copy of the repository in your own GitHub account.
3. Clone the repository locally, by calling `git clone https://github.com/naceurCRAAG/NeuralPDE.jl/` in a directory of your choice.
4. Open The repository in your favourite IDE (vscode is a good one for julia).
5. Create a new branch to work on, it makes rebasing (re integrating changes from the upstream repo that others have made while you were working) easier if you are not working on the master branch.
6. Make the desired changes to the README file. You can use GitHub's markdown syntax to format the text and add images.
7. Once you have finished making the changes, go to the git tab, stage your changes and add a brief description of the changes you made in the "Commit changes" section. Hit "Commit", then sync the changes with your remote fork.
8. Navigate back to the original repository and click on the "New pull request" button. (under the pull requests (PRs) tab)
9. Select your forked repository and the branch containing the changes you made to the README file.
10. Add a brief description of the changes you made in the "Open a pull request" section.
11. Click on the "Create pull request" button to submit your changes to the original repository.
12. Wait for the repository owner to review your changes and merge them into the original repository. Check back regularly in case the reviewer has requested any changes.
13. Congrats, you've just contributed to open source, welcome to the club!

Note: It's a good practice to communicate with the repository owner or maintainers to discuss the changes you want to make before submitting a pull request. This can help avoid conflicts and ensure that your changes are in line with the project's goals.

**Comment by naceurCRAAG** [comment]
Hello Alex :)

Happy to read you :) Thank you so much for your kindness! I will try to do what you advise.

Naceur

**Comment by xtalax** [comment]
Let me know if you get stuck, you can also find me on the slack :)

**Comment by naceurCRAAG** [comment]
Yes ! I will let you know even if I succeed. I try this tomorrow since here it is too late this evening.

**Comment by naceurCRAAG** [comment]
Hello Alex

I did as you explained to me to do
Could you please have a look to it ?
it is here
https://github.com/naceurCRAAG/NeuralPDE.jl/commit/cf701812e6b992f49a34c9dab6ce0a1675c80f03

Have A Nice Day

Naceur

**Comment by xtalax** [comment]
Looks good, please submit a pull request to this repo

**Comment by naceurCRAAG** [comment]
Thanks Alex
How can I submit a pull request ?

**Comment by dstrbad** [comment]
Hi @naceurCRAAG here's the link that should help you, https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by naceurCRAAG** [comment]
Dear Dejan

I did what you taught me 
But I am not sure if it works 
Could you please check if I succeeded 

Naceur

**Comment by dstrbad** [comment]
> Dear Dejan
> 
> I did what you taught me But I am not sure if it works Could you please check if I succeeded
> 
> Naceur

PRs are visible here:
https://github.com/SciML/NeuralPDE.jl/pulls

Your is as you know already accepted:
https://github.com/SciML/NeuralPDE.jl/commit/0f3019547c00a4db4b405f2538e95ed97470b47a
